src/components/Home.react.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Home = () => {
```

Comment:
Yeah these are functional stateless components. They are basically a way to express React components as a pure function of props -> UI state.

I think it's a good idea to use functional stateless components wherever possible. I think they're really clean and it prevents you from getting lazy and needlessly putting state in a component.

Check out [This link](https://medium.com/@housecor/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc#.spofl8su0)